universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -106,6 +106,7 @@ export class ULIFrameProvider {
 
   static create(network: Network, sdkConfig?: Record<string, any>, config = DEFAULT_CONFIG) {
```

Comment:
I think we should change config for configOverrides and merge it with DEFAULT_CONFIG then